### ISSUE TITLE
Chat content rendering is abnormal.

### ISSUE THREAD

**Issue by Caojiahao-Coder** [anse-app/anse#43]
### How is Anse deployed?

Docker

### Describe the bug
> If the question content starts with a code, the conversation message won't render correctly.

#### Input
![image](https://github.com/anse-app/anse/assets/68038353/155af586-ac5d-4953-9799-749a4e391045)

#### Output
![image](https://github.com/anse-app/anse/assets/68038353/3c3c6a17-118f-4531-91eb-d2df7174aca8)

#### Docker log
```
2023-05-25 09:54:12 callProviderHandler {
2023-05-25 09:54:12   conversationId: 'temp',
2023-05-25 09:54:12   conversationType: 'chat_single',
2023-05-25 09:54:12   botId: 'temp',
2023-05-25 09:54:12   globalSettings: {
2023-05-25 09:54:12     baseUrl: 'https://api.openai.com',
2023-05-25 09:54:12     model: 'gpt-3.5-turbo-0301',
2023-05-25 09:54:12     maxTokens: 2048,
2023-05-25 09:54:12     temperature: 0.7,
2023-05-25 09:54:12     top_p: 1,
2023-05-25 09:54:12     apiKey: '****'
2023-05-25 09:54:12   },
2023-05-25 09:54:12   botSettings: {},
2023-05-25 09:54:12   prompt: 'Summarize a short and relevant title of input with no more than 5 words.\n' +
2023-05-25 09:54:12     'Rules:\n' +
2023-05-25 09:54:12     '1. Must use the same language as input.\n' +
2023-05-25 09:54:12     '2. Output the title directly, do not add any other content.\n' +
2023-05-25 09:54:12     'The input is:\n' +
2023-05-25 09:54:12     '<div class="h-100%  flex flex-col color-base">\n' +
2023-05-25 09:54:12     '如何解释这行代码',
2023-05-25 09:54:12   messages: []
2023-05-25 09:54:12 }
```

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by ddiu8081** [comment]
fixed in https://github.com/anse-app/anse/commit/737ed7c456da52851a8517664a42a3a74ead9963

**Comment by Caojiahao-Coder** [comment]
> fixed in [737ed7c](https://github.com/anse-app/anse/commit/737ed7c456da52851a8517664a42a3a74ead9963)

## Sorry the bug isn't fixed in Docker Image (V1.1.5->latest).

> Chrome Console has no error output.

### Input
![1685064613758](https://github.com/anse-app/anse/assets/68038353/b14d9f25-dbf4-41bc-84af-f7b1962d2c4d)
### Output
![1685064621829](https://github.com/anse-app/anse/assets/68038353/0c6c509e-f79e-4273-9b4f-dc25b2b323c5)
### Docker Image
![image](https://github.com/anse-app/anse/assets/68038353/7db317db-9cfd-425f-b239-7f1215f1b889)
